src/components/maps/DataMap/DataMap.js

```diff
@@ -84,28 +84,28 @@ class DataMapDisplay extends React.Component {
       layerSpec: this.props,
       xy: event.layerPoint,
     })
-    .then(response => {
-      const layerInfo = xml2js(response.data, {
-        compact: true,
+      .then(response => {
+        const layerInfo = xml2js(response.data, {
+          compact: true,
+        });
+        const value = layerInfo.foo;
```

Comment:
I know you didn't name this, just formatted, the code, but "foo" is not a terribly enlightening attribute name.